share/completions/git.fish

```diff
@@ -1668,6 +1668,7 @@ complete -f -c git -n '__fish_git_using_command range-diff' -l no-dual-color -d
 complete -f -c git -n __fish_git_needs_command -a push -d 'Update remote refs along with associated objects'
 complete -f -c git -n '__fish_git_using_command push; and not __fish_git_branch_for_remote' -a '(__fish_git_remotes)' -d 'Remote alias'
 complete -f -c git -n '__fish_git_using_command push; and __fish_git_branch_for_remote' -a '(__fish_git_branches)'
+complete -f -c git -n '__fish_git_using_command push; and __fish_git_branch_for_remote' -ka '(__fish_git_tags)' -d Tag
```

Comment:
Looks like -k preserves the relative order of tags, but they are still interleaved with branches, which weird (but acceptable).
I'd flip these two lines and add another -k so we always get (local) branches first.

	complete -f -c git -n '__fish_git_using_command push; and __fish_git_branch_for_remote' -ka '(__fish_git_tags)' -d Tag
	complete -f -c git -n '__fish_git_using_command push; and __fish_git_branch_for_remote' -ka '(__fish_git_branches)'

Reply:
Done. It looks much better this way IMO :smiley: